docs/USAGE.md

```diff
@@ -106,9 +79,9 @@ The client CLI will also expect to find the daemon at `http://localhost:8042`.
 To configure a custom endpoint address, refer to the `[client]` settings on the
 [env-example.toml](../env-example.toml) file at the root of this repo.
 
-## Running the tests locally with Testground
+## Running test plans locally with Testground
```

Comment:
What's actually run is a test case, though. Suggest fixing it here and in the prose.

Reply:
I am just being a bit explicit with saying `test plan`, so that it is clear we are referring to the plans, and not for example to Testground's tests.